extensions/suspend/js/src/forum/extend.ts

```diff
@@ -0,0 +1,11 @@
+import Extend from 'flarum/common/extenders';
+import User from 'flarum/common/models/User';
+import Model from 'flarum/common/Model';
+
+export default [
+  new Extend.Model(User)
+    .attribute<boolean>('canSuspend')
+    .attribute<Date, string>('suspendedUntil', Model.transformDate)
+    .attribute<string>('suspendReason')
+    .attribute<string>('suspendMessage'),
```

Comment:
Perhaps some of these should be nullable?

Reply:
true